### PR TITLE
fix imgexport couldn't find rootimg.gz

### LINKED ISSUE #1903: imgexport failed because couldn't find rootimg.gz file
Report this issue from Lab Service.  when run imgexport command:
````
# imgexport Prhels73snapshot3DL1637a
Exporting Prhels73snapshot3DL1637a to /root/tmp...
Error: Couldn't find rootimg (rootimg.gz) for  Prhels73snapshot3DL1637a
`````
and rootimgdir has:
`````
# ls /install/netboot/rhels73/Prhels73snapshot3DL1637a/
total 2222536
-rw-------  1 root root   41838230 Sep 27 03:34 initrd-stateless.gz
-rw-------  1 root root   39135600 Sep 27 03:35 initrd-statelite.gz
-rwxr-xr-x  1 root root   19008920 Sep 27 03:24 kernel
dr-xr-xr-x 21 root root       4096 Sep 27 03:37 rootimg
-rw-r--r--  1 root root 2175881332 Sep 27 03:41 rootimg.cpio.gz
```````

checking imgport.pm, there are hard coded of "rootimg.gz" file names.  If we change name to rootimg.cpio.gz,   we should take care of those hard coded names.

### PR DESCRIPTION
**Pull request by penguhyang**
fix #1903

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/imgport.pm [hunk 1]
```diff
@@ -631,7 +644,7 @@ sub get_files {
                 }
 
                 unless ($rootimg) {
-                    $callback->({ error => ["Couldn't find rootimg (rootimg.gz) for  $imagename"], errorcode => [1] });
+                    $callback->({ error => ["Couldn't find rootimg(rootimg.cpio.xz or rootimg.cpio.gz or rootimg.tar.xz or rootimg.tar.gz) for $imagename"], errorcode => [1] });
```
Comment (immarvin) [inline xCAT-server/lib/xcat/plugins/imgport.pm]: “could not find compressed rootimg for ”

File: xCAT-server/lib/xcat/plugins/imgport.pm [hunk 2]
```diff
@@ -599,22 +599,35 @@ sub get_files {
...
                 if ($rootimgdir) {
                     if (-f "$rootimgdir/initrd-stateless.gz") {
                         $ramdisk = "$rootimgdir/initrd-stateless.gz";
                     }
                     if (-f "$rootimgdir/kernel") {
                         $kernel = "$rootimgdir/kernel";
                     }
-                    if (-f "$rootimgdir/rootimg.gz") {
-                        $rootimg = "$rootimgdir/rootimg.gz";
+                    if (-f "$rootimgdir/rootimg.cpio.xz") {
```
Comment (immarvin) [inline xCAT-server/lib/xcat/plugins/imgport.pm]: rootimg.gz should also be looked for for backward compatibility, should be lowest priority

Comment (penguhyang) [inline xCAT-server/lib/xcat/plugins/imgport.pm]: fixed.

File: xCAT-server/lib/xcat/plugins/imgport.pm (hunk 1 above)
Comment (penguhyang) [inline xCAT-server/lib/xcat/plugins/imgport.pm]: fixed

**Comment by immarvin** [comment]
I am ok with the change, and I think the influence of this PR is restricted in imgexport/imgimport, so I agree to merge it

**Pull request merged**